pkg/api/http_server.go

```diff
@@ -60,7 +60,7 @@ type HTTPServer struct {
 	RenderService       rendering.Service        `inject:""`
 	Cfg                 *setting.Cfg             `inject:""`
 	HooksService        *hooks.HooksService      `inject:""`
-	CacheService        *cache.CacheService      `inject:""`
+	CacheService        *localcache.CacheService `inject:""`
```

Comment:
why local cache? is the interface not a general one that can be backed by remote redis cache?

Reply:
its not general. I think the usage pattern is different and should be explicit. 
I renamed it to make it more clear that's it's local and in memory. 

Choosing to cache stuff in memory or serialized remotely is two different actions IMO.

Reply:
is it in memory, as so this is not the same cache that is backed by the database (that has lock issues?) Is this the in memory cache?

Reply:
Oh, yea. Sorry for not being explicit about that. 

This is the in-memory only cache. 
the cached backed by the database is named `remotecache` so naming this `localcache` felt more explicit.

Reply:
@torkelo anything else to consider before we merge this?

Reply:
@bergquist may affect enterprise datasource permissions maybe without having looked into the details of the changes

Reply:
@marefr yes! Ive talked to Leo about it and I will open A pr fixing this in the enterprise so we can merge them at the same time.